[3299][ADD] purchase_order_line_product_type (#20)
qrtl/axls-custom: purchase_product_type_info/readme/DESCRIPTION.rst

> This module is created to displays the product type information on purchase order line.

Reviewer: ```suggestion
This module displays product type in lines of purchase order form.
```
